bc_obps/reporting/service/report_activity_save_service.py

```diff
@@ -222,4 +226,29 @@ def save_emission(
         report_emission.set_create_or_update(self.user_guid)
         EmissionCategoryMappingService.apply_emission_categories(report_source_type, report_fuel, report_emission)
 
+        self.save_methodology(report_emission, emission_data['methodology'])
+
         return report_emission
+
+    def save_methodology(
+        self,
+        report_emission: ReportEmission | None,
```

Comment:
the database model enforces the foreign key to ReportEmission to not be null, so the `None` case doesn't exist
```suggestion
        report_emission: ReportEmission,
```